_includes/known-issues-alert.html

```diff
@@ -0,0 +1,22 @@
+{% if issues.alert-type == "info" %}
+  <div class="usa-alert margin-bottom-3 usa-alert--info">
+    <div class="usa-alert__body">
+      <h2 class="usa-alert__heading">
+        Recent testing has revealed some issues with this component.
```

Comment:
Recommendation from Jacline: Remove the word "Recent" from this copy since it is already 2 months old and it won't stay "recent" for long.
```suggestion
        Testing has revealed some issues with this component.
```